packages/common/index.ts

```diff
@@ -312,6 +312,8 @@ export interface Queue {
  * @param isOpen - A queue is open if it has staff and is not disabled.
  * @param config - A JSON object that contains the configuration for the queue. Contains stuff like tags, tasks, etc.
  */
+export type QueueTypes = 'online' | 'hybrid' | 'inPerson'
```

Comment:
perhaps a better name for this is QueueLocations?

Also, you might want to move this since it's right below QueuePartial's jsdoc

Reply:
Whoops, didn't notice the jsdoc hahah.